### PR TITLE
Added workflow test for metaquantome data creation tutorial

### PR DESCRIPTION
**Pull request by malloryfreeberg**
# Summary of changes

- Added workflow test
- Updated workflow

### CODE REVIEW

File: topics/proteomics/tutorials/metaquantome-data-creation/workflows/main_workflow.ga [hunk 1]
```diff
@@ -11,15 +11,15 @@
...
+            "label": "Input_collection",
+            "name": "Input_collection",
             "outputs": [],
             "position": {
                 "left": 267.953125,
                 "top": 714.65625
             },
             "tool_id": null,
-            "tool_state": "{\"optional\": false, \"collection_type\": \"list\"}",
+            "tool_state": "{\"name\": \"good_outcome\"}",
```
Comment (bgruening) [inline topics/proteomics/tutorials/metaquantome-data-creation/workflows/main_workflow.ga:22]: this looks wired, do you know why this is needed?

File: topics/proteomics/tutorials/metaquantome-data-creation/workflows/main_workflow.ga [hunk 2]
```diff
@@ -40,18 +40,18 @@
...
+            "label": "ProteinDB_cRAP.fasta",
+            "name": "ProteinDB_cRAP.fasta",
             "outputs": [],
             "position": {
                 "left": 244.40625,
                 "top": 921.640625
             },
             "tool_id": null,
-            "tool_state": "{\"optional\": false}",
+            "tool_state": "{\"name\": \"ProteinDB_cRAP.fasta\"}",
```
Comment (bgruening) [inline topics/proteomics/tutorials/metaquantome-data-creation/workflows/main_workflow.ga:54]: Do you really need to change the tool state.

File: topics/proteomics/tutorials/metaquantome-data-creation/workflows/main_workflow.ga (hunk 1 above)
Comment (malloryfreeberg) [reply topics/proteomics/tutorials/metaquantome-data-creation/workflows/main_workflow.ga:22]: This is a copy/paste mistake from the non-collections workflow test example. Fixing now.

File: topics/proteomics/tutorials/metaquantome-data-creation/workflows/main_workflow.ga (hunk 2 above)
Comment (malloryfreeberg) [reply topics/proteomics/tutorials/metaquantome-data-creation/workflows/main_workflow.ga:54]: I'm going by the example [here](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-technical/tutorial.html#testing-the-workflow-recommended) which indicates to align the inputs name, label, name, and tool_state name:

```
{
    "id": 0,
    "input_connections": {},
    "inputs": [
        {
            "description": "",
            "name": "good_outcome"
        }
    ],
    "label": "good_outcome",
    "name": "good_outcome",
    "outputs": [],
    "position": {
        "left": 10,
        "top": 10
    },
    "tool_id": null,
    "tool_state": "{\"name\": \"good_outcome\"}",
    "tool_version": null
}
```

**Review by bgruening — APPROVED** [review]

**Pull request merged**